connect/runtime/src/main/java/org/apache/kafka/connect/converters/BooleanConverterConfig.java

```diff
@@ -25,13 +25,13 @@
  */
 public class BooleanConverterConfig extends ConverterConfig {
```

Comment:
Checked the code and seems only NumberConverter has a config applied to IntegerConverter, LongConverter and FloatConverter. Classes like ByteArrayConverter does not have a config. Also seems this config does not have any special props, is there a reason to add it?

Reply:
We need a concrete class that extends the abstract `org.apache.kafka.connect.storage.ConverterConfig`. 

Unlike `ByteArrayConverter` - which is just a pass-through converter that performs no serialization/deserialzation, the serializer/deserializer for `BooleanConverter` needs to know if the field being serialized is the `key`, and this logic is implemented by the parent `ConverterConfig` class.